metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java

```diff
@@ -43,68 +42,30 @@
 
```

Comment:
unit tests for this file are missing.

Reply:
As we discussed, we allow this in this iteration